frontends/smv_node.h

```diff
@@ -0,0 +1,35 @@
+#ifndef NODE_H
+#define NODE_H
+#pragma once
+#include <stdio.h>
+#include <iostream>
+#include <map>
+#include <string>
+#include <unordered_map>
+
+#include "assert.h"
+#include "exceptions.h"
+#include "smt-switch/smt.h"
+
+struct node {
+      smt::Term tm;
+      smt::Sort st;
+
+      node (smt::Sort s){
```

Comment:
is it correct: a node is either a term or a sort?
if yes, how do we know after it is created

Reply:
Hi Ahmed,
Thanks for your comments! A node is either a term or a sort. I want to use a string to denote which type it is. Is this method enough or is there a better implementation?

Reply:
string is too heavy for this purpose. maybe use enum or a boolean instead.